application-admintools-ui/src/main/resources/AdminTools/Code/Translations.xml

```diff
@@ -36,23 +36,40 @@
   <syntaxId>plain/1.0</syntaxId>
   <hidden>true</hidden>
   <content>##Admin section
+AdminTools.Code.ConfigurationClass_excludedLines=Excluded lines
+AdminTools.Code.ConfigurationClass_excludedLines.hint=Lines containing sensitive info in xwiki.cfg and xwiki.properties files
 AdminTools.Code.ConfigurationClass_serverLocation=Server location
 AdminTools.Code.ConfigurationClass_serverLocation.hint=Path to the server location
 admin.admintools.description=Configure Admin Tools Application
+adminTools.dashboard.modal.button.close=Close
+adminTools.dashboard.modal.download.button=Download selected files
+adminTools.dashboard.modal.download.date.from=From:
+adminTools.dashboard.modal.download.date.to=To:
+adminTools.dashboard.modal.download.provided=Configuration info
+adminTools.dashboard.modal.download.title=Select Files to Download
+adminTools.dashboard.modal.logs.button=View logs
+adminTools.dashboard.modal.logs.selection=Insert the number of lines to show:
```

Comment:
`adminTools.dashboard.logs.modal.nLines.label=Number of extracted lines`
No need for the `:`, since this what a label does, links the input to a text